### ISSUE TITLE
multiple commands in CIBW_BEFORE_BUILD on windows 

### ISSUE THREAD

**Issue by louisabraham** [joerick/cibuildwheel#289]
I cannot get CIBW_BEFORE_BUILD to execute multiple commands on windows. I tried separating them with `;` but get an error (https://travis-ci.org/louisabraham/pydivsufsort/jobs/658762195).

**Comment by joerick** [comment]
Anyone know enough about Windows shell to do this? We run the command using:

```python
subprocess.run(['cmd', '/E:ON', '/V:ON', '/C', '<command>'])
```

**Comment by joerick** [comment]
According to https://ss64.com/nt/cmd.html, it will support joining commands with '&&' if surrounded with double-quotes. If #290 works that should make this pretty simple to add :)

**Comment by louisabraham** [comment]
nice! It could be useful to add to the documentation.

**Comment by joerick** [comment]
Oh, I mean we'd surround with double-quotes internally to cibuildwheel, not that users should do that

**Comment by joerick** [comment]
Whoops, fumbled the close button

**Comment by louisabraham** [comment]
The meaning of the doc is not very clear: I think it's the commands that should be surrounded by quotes, not the ampersand.

**Comment by joerick** [comment]
This is fixed in #293 - no quotes required!
